popup/index.tsx

```diff
@@ -1,26 +1,301 @@
-import { useState } from "react"
+import * as ethers from "ethers"
+import { useCallback, useEffect, useState, type MouseEvent } from "react"
+
+import { BackgroundDirectMessenger } from "~packages/messenger/background/direct"
+import { WaalletContentProvider } from "~packages/provider/waallet/content/provider"
 
 import "~style.css"
 
+const provider = new ethers.BrowserProvider(
+  new WaalletContentProvider(new BackgroundDirectMessenger())
+)
+
 function IndexPopup() {
-  const [data, setData] = useState("")
+  const [accountIndex, setAccountIndex] = useState<number>(0)
+  const [accounts, setAccounts] = useState<string[]>([""])
+  const [balances, setBalances] = useState<bigint[]>([0n])
```

Comment:
nit: These two fields could be simplify to single value? We don't need to consider case of multiple accounts at this moment, right?